js/jsInteresting.py

```diff
@@ -115,7 +115,14 @@ def __init__(self, options, runthis, logPrefix, inCompareJIT):
         crashInfo = CrashInfo.CrashInfo.fromRawCrashData(out, err, pc, auxCrashData=auxCrashData)
 
         createCollector.printCrashInfo(crashInfo)
-        if not isinstance(crashInfo, CrashInfo.NoCrashInfo):
+        if isinstance(crashInfo, CrashInfo.NoCrashInfo):
+            # Note that looking out for the Assertion failure message is highly SpiderMonkey-specific
+            if 'Assertion failure: ' in str(crashInfo.rawStderr) or \
+                    'Segmentation fault' in str(crashInfo.rawStderr) or \
+                    'Bus error' in str(crashInfo.rawStderr):
+                lev = max(lev, JS_NEW_ASSERT_OR_CRASH)
+            pass  # We only care about crashes and assertion failures on shells with no symbols
```

Comment:
Actually shouldn't this be:

    if not isinstance(crashInfo, CrashInfo.NoCrashInfo) or \
            'Assertion failure: ' in str(crashInfo.rawStderr) or \
            'Segmentation fault' in str(crashInfo.rawStderr) or \
            'Bus error' in str(crashInfo.rawStderr):
        lev = max(lev, JS_NEW_ASSERT_OR_CRASH)

Which I think would be clearer. What do you think?